sphinxext/opengraph/__init__.py

```diff
@@ -134,7 +134,9 @@ def get_tags(
             image_url_parsed = urlparse(image_url)
             if not image_url_parsed.scheme:
                 # Relative image path detected. Make absolute.
-                image_url = urljoin(config["ogp_site_url"], image_url_parsed.path)
+                image_url = urljoin(
+                    config["ogp_site_url"], image_url_parsed.path.split("../")[-1]
```

Comment:
You can use str.rpartition to do this, which is both more efficient and likely conveys intent more clearly.